application/engine/services/autopilot_daemon.py

```
@@ -338,6 +338,13 @@ async def _handle_act_planning(self, novel: Novel):
 
         target_act = next((n for n in act_nodes if n.number == target_act_number), None)
 
+        # 预计算结构参数（确保在所有代码路径中都可用）
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)